lib/build.gradle

```diff
@@ -41,6 +41,7 @@ android {
         }
     }
     compileOptions {
+        coreLibraryDesugaringEnabled true
```

Comment:
What's this about?

Reply:
Concrete usage in this PR: https://github.com/nextcloud/Android-SingleSignOn/pull/410/files#diff-61442b83bbc4d761051337c72c535ce0e5a14da8c3aa3c5767f76bef95b30652R78

```java
private static final String[] ACCOUNT_TYPES = Arrays.stream(FilesAppType.values()).map(a -> a.accountType).toArray(String[]::new);
```

Reply:
Totally fine, I was not aware of this.